extensions/fhir/README.adoc

```diff
@@ -0,0 +1,9 @@
+== Camel Quarkus FHIR extention
```

Comment:
@jamesnetherton today's patch started a new convention that the extension doc pages live in `docs/modules/ROOT/pages/extensions` . What do you think about transforming this README to such a page?

Reply:
done